hbase-server/src/main/java/org/apache/hadoop/hbase/replication/NamespaceTableCfWALEntryFilter.java

```diff
@@ -18,27 +18,17 @@
 
 package org.apache.hadoop.hbase.replication;
 
-import java.util.List;
-import java.util.Map;
 import org.apache.hadoop.hbase.Cell;
 import org.apache.hadoop.hbase.CellUtil;
 import org.apache.hadoop.hbase.TableName;
-import org.apache.hadoop.hbase.util.Bytes;
 import org.apache.hadoop.hbase.wal.WAL.Entry;
 import org.apache.hadoop.hbase.wal.WALEdit;
 import org.apache.yetus.audience.InterfaceAudience;
 
 /**
- * Filter a WAL Entry by the peer config: replicate_all flag, namespaces config, table-cfs config,
```

Comment:
Is the behaviour described here not valid anymore? I couldn't see any detailed explanation in ReplicationPeerConfig#needToReplicate.

Reply:
The behaviour described here is same with what ReplicationPeerConfig#needToReplicate does, so I just deleted the description.